web/controllers/registration_controller.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Yggdrasil.RegistrationController do
+  use Yggdrasil.Web, :controller
+
+  import Ecto.Changeset, only: [put_change: 3]
+  import Comeonin.Bcrypt, only: [hashpwsalt: 1]
```

Comment:
__bows__ thank you, thank you __bows__ (fake crowd goes wild)

I'd like to thank Elixir and Phoenix for the opportunity for being here tonight and most of all I would like to give a shout out to Comeonin dot Bcrypt because without you `Yay, bcrypt!` would not be a thing. Also would like to thank the blogs where I copied this from and sorry you guys couldn't be here tonight.

Good night every body